nextflow.config

```diff
@@ -89,4 +89,4 @@ profiles {
         params.clinical_url  = 'https://clinical.dev.argo.cancercollaboratory.org'
         params.schema_url    = 'https://submission-song.rdpc-dev.cancercollaboratory.org/schemas/sequencing_experiment'
     }
-}
```

Comment:
Please remove the default setting from config, since we will provide score_cpu and score_mem as the container cpu and mem. 
```
    withName: scoreUpload {
        cpus = 4
        memory = 10.GB
    }
```